tests/doc_scan_service/session/create/session.specification.builder.test.js

```diff
@@ -64,6 +64,10 @@ describe('SessionSpecificationBuilder', () => {
       .withFilter(documentFilter)
       .build();
 
+    const subjectDescriptor = {
```

Comment:
Your test is absolutely valid, yet, I would prefer to extract the `withSubject` to its own separated test.
That is cause `withSubject` is not meant to be used outside of the TFIP context (as a matter of fact, the IDV API could throw an error, at best ignore it).
This test file initially was 1 test that was adding all the options in one session builder call, but, then we introduce more granular tests as we added options.